model/jpa/src/main/resources/META-INF/jpa-changelog-13.0.0.xml

```diff
@@ -42,4 +42,9 @@
         <dropForeignKeyConstraint baseTableName="DEFAULT_CLIENT_SCOPE" constraintName="FK_R_DEF_CLI_SCOPE_SCOPE"/>
         <dropForeignKeyConstraint baseTableName="CLIENT_SCOPE_CLIENT" constraintName="FK_C_CLI_SCOPE_SCOPE"/>
     </changeSet>
+
+    <changeSet author="keycloak" id="json-string-accomodation">
```

Comment:
Is it possible to change this to use `newDataType="NCLOB"` ? I've actually added this change to my branch https://github.com/mposolda/keycloak/tree/KEYCLOAK-16805-client-policies-db-change (See the last commit). I've tested that updated change with all our supported databases nad it is fine.

The reason for that change is, that NVARCHAR(65535) doesn't work on some databases (At least Oracle has limit 4000 characters for the VARCHAR and NVARCHAR). Also we use CLOB or NCLOB for some similar columns in the DB where is JSON supposed to be saved.

Reply:
@mposolda Thank you for your proposal. I would like to base your branch adding NCLOB support based on this PR's branch and use this from now on.

Reply:
Ok, that should help!